apps/wallet/src/background/accounts/zk/utils.ts

```diff
@@ -105,8 +105,7 @@ export async function zkLogin({
 	return jwt;
 }
 
-// TODO: update when we have the final production url
-const saltRegistryUrl = 'http://salt.api-devnet.mystenlabs.com';
+const saltRegistryUrl = 'https://salt.api.mystenlabs.com/get_salt';
```

Comment:
This currently is not ready but we need to publish the wallet as early as possible to account for the review time

Reply:
context [here](https://mysten-labs.slack.com/archives/C05Q4FNS31A/p1695337201866049?thread_ts=1695336036.540039&cid=C05Q4FNS31A)